### DISCUSSION TITLE
Molecular networking in R - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#445] [General]
# Molecular networking in R - Miao Yu | 于淼

https://yufree.cn/en/2023/06/25/molecular-networking-in-r/

**Comment by yxhuang** [comment]
偶然搜到您的blog，风格跟Xie Yihui的好像~也是做统计的大神，收藏了，之后常来逛~

**Reply by yxhuang** [reply]
为啥会被划掉- -

**Reply by yufree** [reply]
因为两个波浪线在markdown里是删除线的标记，前后两个波浪线之间的内容会被标记为删除。

**Reply by yihui** [reply]
当年 Markdown 语法在标准化的时候要是征求一下中国用户的意见，就不会总是出这种翻车事故了。中文一卖萌，就会被删除。切记以后要用～胖乎乎的～～全角～～～波浪线～～～～
